src/Apps/Artist/routes.tsx

```diff
@@ -231,6 +205,57 @@ export const routes: RouteConfig[] = [
           return filterParams
         },
       },
+      {
+        path: "auction-results",
+        getComponent: () => AuctionResultsRoute,
+        prepare: () => {
+          AuctionResultsRoute.preload()
+        },
+        displayNavigationTabs: true,
+        query: graphql`
+          query routes_AuctionResultsQuery($artistID: String!) {
+            artist(id: $artistID) {
+              ...AuctionResults_artist
+            }
+          }
+        `,
+      },
+
+      // FIXME: Remove the following unused routes
+      {
+        path: "cv",
```

Comment:
No preload for these?

Reply:
No longer used

Reply:
Hmmm, I think I'm missing something. These pages still exist on the artist page, they're just behind links instead of tabs. That means the routes are still needed, right?

Reply:
Oh, i didn't realize these were being used outside of the tabs. If that's the case then yup, we should load them as well in a similar fashion. Will update that now.

Reply:
Done